apps/meteor/server/modules/watchers/watchers.module.ts

```diff
@@ -268,7 +268,7 @@ export function initWatchers(watcher: DatabaseWatcher, broadcast: BroadcastCallb
 		}
 	});
 
-	watcher.on<ISetting>(Settings.getCollectionName(), async ({ clientAction, id, data, diff }) => {
+	watcher.onCollection<ISetting>(Settings.getCollectionName(), async ({ clientAction, id, data, diff }) => {
 		if (diff && Object.keys(diff).length === 1 && diff._updatedAt) {
```

Comment:
Random but I think this check should be on all listeners by default

Reply:
hmm.. I'll take a look at it to see if it makes sense.. thx for pointing it out